src/components/filter/toggle/index.tsx

```diff
@@ -30,15 +30,22 @@ export class SmoothlyFilterToggle {
 					const criteria = !this.not ? r : r instanceof selectively.Not ? r.criteria : undefined
 					return value == ""
 						? !(criteria instanceof selectively.Property && criteria.name == key)
-						: criteria?.is({ [key]: value }) &&
-								criteria instanceof selectively.Property &&
-								this.properties[criteria.name] &&
-								criteria.criteria instanceof selectively.Is
+						: this.isCriteria(criteria, key, value)
 				})
 			)
 		} else
 			this.active = false
 	}
+	private isCriteria(criteria: selectively.Rule | undefined, key: string, value: string): unknown {
```

Comment:
Why is the return value "unknown" instead of "boolean"?

Reply:
Good question :laughing: